.travis.yml

```diff
@@ -1,7 +1,5 @@
+sudo: false
 language: python
-python:
-  - "2.7"
-install: 
-  - "pip install PyYAML"
-script: 
-  - "./scripts/git-hooks/pre-commit"
+install:
+  - pip install tox-travis
```

Comment:
Yeah, it looks like we will need to tell Travis about all three Python versions; otherwise, it won't install 3.5. I think tox-travis is supposed to automatically invoke `tox` with the right arguments for each env if we do that (so we can just put `tox` in the Travis file).